### DISCUSSION TITLE
Good pack Mypaint

### DISCUSSION THREAD

**Discussion by FoxBoxFox** [opentoonz/opentoonz#6748] [Brushes]
Good pack Mypaint
![image](https://user-images.githubusercontent.com/39651284/40577072-30d22ce6-6108-11e8-959f-8253cb884bf7.png)

https://senlinos.github.io/post/mypaint-brushes-for-gimp-2.10-by_senlinos/ 

This is a new pack.

**Comment by RodneyBaker** [comment]
Thanks for the heads up @FoxBoxFox 

I just ran a quick test and... looking good!
I didn't see anything that didn't work in OpenToonz.
I see also that the CC license allows for full use and distribution.

We'll have to confer with the folks that incorporated the last brush set and see if we can get this added too.

It is getting to the point where there are so many brushes we may need to consider adjusting how the brushes are organized and presented to the user.

**Comment by artisteacher** [comment]
If more brushes are added, there definitely needs to be a way to organize them better. It would be great to have the brushes organized by type & to be able to enable/disable the different brush groups. It would also be great for frequently used brushes to display at the top - or perhaps to have a favoriting system where brush favorites display at the top.

**Comment by beeheemooth** [comment]
@artisteacher My Paint brushes can be organized in Style Editor, creating several styles and selecting for each of them your brush with your settings and even with your own color, which will not change if you switch to a different style. Between styles, you can switch with the keys 1-9, and "Tab" to scroll through the ten styles. Selected styles are saved separately for each project, and I find it very convenient.

**Comment by artisteacher** [comment]
Do you mean the level palette? I know, I just would rather have them organized in the style editor. I like to create a palette of colors & change between brushes regularly.

**Comment by RodneyBaker** [comment]
The link to this MyPaint brush pack is broken with a 404 page.
@FoxBoxFox can you update your link?

Also:  Recommend closing this and linking to this issue via the primary MyPaint PR.

**Comment by RodneyBaker** [comment]
Closing but let's remember to revisit this one moving into v1.5.
REF:  opentoonz/opentoonz_docs#136

Reopened for consideration of mypaint update in next release.

**Comment by RodneyBaker** [comment]
There is currently consideration to include this brush set for v1.7
Feedback will be very welcome!

See opentoonz/opentoonz#4600 for PR adding this brush set.
